static/src/javascripts/projects/common/modules/crosswords/helpers.js

```diff
@@ -0,0 +1,360 @@
+// @flow
+import { constants } from 'common/modules/crosswords/constants';
+import findIndex from 'lodash/arrays/findIndex';
+import flatten from 'lodash/arrays/flatten';
```

Comment:
```js
const flatten = list => list.reduce(
    (a, b) => a.concat(Array.isArray(b) ? flatten(b) : b), []
);
```

Reply:
I like the idea of reducing dependencies, but are we going to reimplement this same function everywhere that currently uses `lodash/arrays/flatten`? And if we apply this logic to all the small `lodash` functions that have not yet been implemented by the vanilla JS API, would we be in danger of building our own `lodash`?